### PR TITLE
✨Call Activity Navigation

### LINKED ISSUE #60: Navigate to 'Called Element'
#### Describe your issue

It would be nice to be able to navigate to the called element when using a 'Call Activity'.
In complex processes there might be the need to use some 'Call Activities' to keep clearly arranged processes. 
To call another bpmn process you need to refer to the processId of the bpmn you want to call.
If you want to edit the bpmn you are calling in the process, you need to know the processes name to which the processId belongs to, navigate to the processDef List and select the process.
So it would be much more comfortable to navigate directly to the called element from the process in which it is called.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### LINKED ISSUE #192: Navigate from call-activity
#### Describe your issue

It would be nice to navigate directly to the referenced process.

Let's say you reference a process via call-activity inside another process. By clicking on the call-activity the property panel should show a button that navigates you to the referenced process. Maybe in a new tab. 

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by Vyperus**
## What did you change?

This branch adds the possibility to navigate directly to the called Process when using a Call Activitiy.

Closes #192 
Closes #60 

## How can others test the changes?

- Checkout this branch & start bpmn-studio
- Create a new process and use a call activity
- See the changes in the property panel section for call activities

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/modules/property-panel/indextabs/general/sections/call-activity/call-activity.ts [hunk 1]
```diff
@@ -1,19 +1,42 @@
+import {IPagination, IProcessDefEntity} from '@process-engine/consumer_client';
+import {inject} from 'aurelia-framework';
+import {Router} from 'aurelia-router';
 import {
+  IBpmnModeler,
   ICallActivityElement,
+  IModdleElement,
```
Comment (alexanderkasten) [inline src/modules/property-panel/indextabs/general/sections/call-activity/call-activity.ts]: IModdleElement is unused

File: src/modules/property-panel/indextabs/general/repository/general.repository.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      };
+      const url: string = `${environment.processengine.routes.processes}/${processDef.id}/updateBpmn`;
+      result = await this.httpClient.fetch(url, options);
+
+    } catch (error) {
+      updateError = error;
+    }
+
+    if (updateError) {
+      throw updateError;
```
Comment (heikomat) [inline src/modules/property-panel/indextabs/general/repository/general.repository.ts]: There is no need to add a try/catch block, of you directly rethrow the error anyway

File: src/modules/property-panel/indextabs/general/repository/general.repository.ts [hunk 3]
```diff
@@ -19,30 +19,16 @@ export class GeneralRepository {
...
+      method: 'post',
+      headers: new Headers({
+        'Content-Type': 'application/json',
+      }),
+      body: JSON.stringify({
+          xml: xml,
+      }),
+    };
+    const url: string = `${environment.processengine.routes.processes}/${processDef.id}/updateBpmn`;
+    return await this.httpClient.fetch(url, options);
```
Comment (heikomat) [inline src/modules/property-panel/indextabs/general/repository/general.repository.ts]: you don't need to `return await`, just return the promise that `this.httpClient.fetch` returns to you.

Doing this also makes the `async`-keyword on this method obsolete

File: src/modules/property-panel/indextabs/general/repository/general.repository.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  private httpClient: HttpClient;
+
+  constructor(consumerClient: ConsumerClient, httpClient: HttpClient) {
+    this.consumerClient = consumerClient;
+    this.httpClient = httpClient;
+  }
+
+  public async getAllProcesses(): Promise<IPagination<IProcessDefEntity>> {
+    const processes: IPagination<IProcessDefEntity> = await this.consumerClient.getProcessDefList();
+    return processes;
```
Comment (heikomat) [inline src/modules/property-panel/indextabs/general/repository/general.repository.ts]: you can shorten this method:
```TypeSCript
  public getAllProcesses(): Promise<IPagination<IProcessDefEntity>> {
    return this.consumerClient.getProcessDefList();
  }
```

**Comment by heikomat** [comment]
Sorry for not requesting all changes at once

**Pull request merged**